content/en/docs/concepts/configuration/secret.md

```diff
@@ -619,10 +625,8 @@ start until all the pod's volumes are mounted.
 Create a secret containing some ssh keys:
 
 ```shell
-kubectl create secret generic ssh-key-secret --from-file=ssh-privatekey=/path/to/.ssh/id_rsa 
-```
-```
---from-file=ssh-publickey=/path/to/.ssh/id_rsa.pub
+kubectl create secret generic ssh-key-secret --from-file=ssh-privatekey=/path/to/.ssh/id_rsa \
```

Comment:
do we need `\` ? we haven't formatted/ done on other commands and so will be good to keep consistency